google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/facet/flexible/FlexibleFacetEditor.java

```diff
@@ -234,15 +234,20 @@ private void showWarnings() {
 
   private void toggleDockerfileSection() {
     boolean enabled = false;
+    boolean visible = true;
```

Comment:
do we even need this variable anymore? it looks like it just gets assigned to enabled